packages/odyssey-react-mui/src/theme/palette.ts

```diff
@@ -16,76 +16,76 @@ import * as Tokens from "@okta/odyssey-design-tokens";
+    white: "#ffffff",
   },
   primary: {
-    lighter: Tokens.ColorPaletteBlue000,
-    light: Tokens.ColorPaletteBlue300,
-    main: Tokens.ColorPaletteBlue500,
-    dark: Tokens.ColorPaletteBlue900,
-    contrastText: Tokens.ColorTextBodyInverse,
+    lighter: Tokens.HueBlue50,
+    light: Tokens.HueBlue300,
+    main: Tokens.HueBlue500,
+    dark: Tokens.HueBlue900,
+    contrastText: Tokens.FontColorBodyInverse,
   },
   secondary: {
     light: "#80c7ca",
-    main: Tokens.ColorPaletteTurquoise500,
+    main: "#096256",
     dark: "#004650",
-    contrastText: Tokens.ColorTextBodyInverse,
+    contrastText: Tokens.FontColorBodyInverse,
   },
   error: {
-    lighter: Tokens.ColorPaletteRed000,
-    light: Tokens.ColorPaletteRed300,
-    main: Tokens.ColorPaletteRed500,
-    dark: Tokens.ColorPaletteRed900,
-    contrastText: Tokens.ColorTextBodyInverse,
+    lighter: Tokens.HueRed50,
+    light: Tokens.HueRed300,
+    main: Tokens.HueRed500,
+    dark: Tokens.HueRed900,
+    contrastText: Tokens.FontColorBodyInverse,
   },
   warning: {
-    lighter: Tokens.ColorPaletteYellow000,
-    light: Tokens.ColorPaletteYellow300,
-    main: Tokens.ColorPaletteYellow500,
-    dark: Tokens.ColorPaletteYellow900,
-    contrastText: Tokens.ColorTextBody,
+    lighter: Tokens.HueYellow50,
+    light: Tokens.HueYellow300,
+    main: Tokens.HueYellow500,
+    dark: Tokens.HueYellow900,
+    contrastText: Tokens.FontColorBody,
   },
   info: {
-    lighter: Tokens.ColorPaletteBlue000,
-    light: Tokens.ColorPaletteBlue300,
-    main: Tokens.ColorPaletteBlue500,
-    dark: Tokens.ColorPaletteBlue900,
-    contrastText: Tokens.ColorTextBodyInverse,
+    lighter: Tokens.HueBlue50,
+    light: Tokens.HueBlue300,
+    main: Tokens.HueBlue500,
+    dark: Tokens.HueBlue900,
+    contrastText: Tokens.FontColorBodyInverse,
   },
   success: {
-    lighter: Tokens.ColorPaletteGreen000,
-    light: Tokens.ColorPaletteGreen300,
-    main: Tokens.ColorPaletteGreen500,
-    dark: Tokens.ColorPaletteGreen900,
-    contrastText: Tokens.ColorTextBodyInverse,
+    lighter: Tokens.HueGreen50,
+    light: Tokens.HueGreen300,
+    main: Tokens.HueGreen500,
+    dark: Tokens.HueGreen900,
+    contrastText: Tokens.FontColorBodyInverse,
   },
   grey: {
-    50: Tokens.ColorPaletteNeutral000,
-    100: Tokens.ColorPaletteNeutral100,
-    200: Tokens.ColorPaletteNeutral200,
-    300: "#c1c1c8",
-    400: "#aaaab4",
-    500: Tokens.ColorPaletteNeutral500,
-    600: Tokens.ColorPaletteNeutral600,
-    700: "#585862",
-    800: "#41414b",
-    900: Tokens.ColorPaletteNeutral900,
+    50: Tokens.HueNeutral50,
+    100: Tokens.HueNeutral100,
+    200: Tokens.HueNeutral200,
+    300: Tokens.HueNeutral300,
+    400: Tokens.HueNeutral400,
+    500: Tokens.HueNeutral500,
+    600: Tokens.HueNeutral600,
+    700: Tokens.HueNeutral700,
+    800: Tokens.HueNeutral800,
+    900: Tokens.HueNeutral900,
     // These are "Accent" colors. MUI's palette matches them to the standard greys.
```

Comment:
Where are these used in MUI? Is there future potential to use these for real accents or brand colors? Fine as-is for now, just wondering.

Reply:
As far as I can tell, MUI doesn't currently use them. They exist on Google's Material, so that may be why they're included. I don't know if MUI 6 will use them.